code/modules/martial_arts/krav_maga.dm

```diff
@@ -138,9 +138,10 @@
 //Krav Maga Gloves
 
 /obj/item/clothing/gloves/color/black/krav_maga
-	var/datum/martial_art/krav_maga/style
 	can_be_cut = FALSE
 	resistance_flags = NONE
+	dyeable = FALSE
```

Comment:
another "balance" change here I think. I think just the krav_maga/sec aren't meant to be dyeable.

Reply:
Can swap it over just to the sec one. Will do it once I get home after my flight